scripts/kurtosis/network_params.json

```diff
@@ -2,8 +2,7 @@
     "mev_type": "full",
     "participants": [
         {
-            "el_client_type": "nethermind",
```

Comment:
Can you please elaborate on that?
Maybe this is some kind of issue which we would raise to Client team to look on in meantime?

Reply:
Blocks are not submitted to the relay when using nethermind client at current fuzz rate. I did not look into details of logs, but my guess is the client is busy processing transactions thus not peering them to builders. Using geth builders do produce blocks so as a quickfix I've decided to use geth

Reply:
I've considered fuzzing builders directly, however I've decided not to modify the architecture (and it would be unfair for one builder of course